### PR TITLE
[gPTP] Increase timeout retries for getting TX timestamp

### PR DESCRIPTION
**Pull request by chaitu236**
For some hardware TX timestamp is returned ~1 second after packet is sent. Not reading the timestamp now causes gPTP to report huge link delays (possibly because the stale timestamp is correlated with next packet).

### CODE REVIEW

**Comment by k-gupta** [comment]
Can you add more context on why you are making this change in the change list?

File: daemons/gptp/common/avbts_message.hpp [hunk 1]
```diff
@@ -125,7 +125,7 @@
 #define PTP_PTPTIMESCALE_BIT 3	/*!< PTPTIMESCAPE bit offset on PTPTIMESCALE byte*/
 
 #define TX_TIMEOUT_BASE 1000 	/*!< Timeout base in microseconds */
-#define TX_TIMEOUT_ITER 6		/*!< Number of timeout iteractions for sending/receiving messages*/
+#define TX_TIMEOUT_ITER 12		/*!< Number of timeout iteractions for sending/receiving messages*/
```
Comment (k-gupta) [inline daemons/gptp/common/avbts_message.hpp:128]: Add comment on why this was changed.

File: daemons/gptp/common/avbts_message.hpp [hunk 2]
```diff
@@ -125,7 +125,7 @@
 #define PTP_PTPTIMESCALE_BIT 3	/*!< PTPTIMESCAPE bit offset on PTPTIMESCALE byte*/
```
Comment (k-gupta) [inline daemons/gptp/common/avbts_message.hpp:125]: This looks like this might be similar change - https://github.com/AVnu/OpenAvnu/pull/728/files

Comment (chaitu236) [reply daemons/gptp/common/avbts_message.hpp:125]: Looks to be the right fix. Opened https://github.com/ni/Open-AVB/pull/42. Closing this.

**Pull request closed without merging**